### PR TITLE
1209542: Surface subscription number in CLI and GUI listings

### PR DESCRIPTION
**Pull request by wottop**
This is informational. It was requested to help the admin make
attaching decisions.

### CODE REVIEW

**Comment by alikins** [comment]
Is there a difference between subscription_number and subscriptionID? ie, why two different naming conventions?

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -2322,6 +2327,7 @@ def _do_command(self):
                                 data['suggested'],
                                 data['service_level'] or "",
                                 data['service_type'] or "",
+                                subNumber,
```
Comment (alikins) [inline src/subscription_manager/managercli.py]: could this be
```
data['subscriptionId'] or "",
```
Without the bits around line 2312-2314?

I think 'data' here should always be populated with a subscriptionsId key in get_available_entitlements?

**Comment by wottop** [comment]
I assumed that the sub id was just called number in the presentation. I am working with some folks to determine this. Review away, but don't merge.

File: src/subscription_manager/managercli.py (hunk 1 above)
Comment (wottop) [inline src/subscription_manager/managercli.py]: I thought it would too, but against the test data, it would error out that way.

**Comment by wottop** [comment]
Requires https://github.com/candlepin/python-rhsm/pull/164

**Pull request closed without merging**